bears/c_languages/GNUIndentBear.py

```diff
@@ -193,7 +194,7 @@ def create_arguments(filename, file, config_file,
 
         # The limit is set to a number which equals to int max in C
```

Comment:
The change you have done is ok, but this doesnt solve the problem.

`max_line_length` is not used after here.
It needs to be set much earlier in the function, before it is used.